pkg/scalers/rabbitmq_scaler.go

```diff
@@ -99,7 +99,13 @@ func parseRabbitMQMetadata(resolvedEnv, metadata, authParams map[string]string)
 			return nil, fmt.Errorf("no apiHost setting given")
 		}
 	} else {
-		if val, ok := authParams["host"]; ok {
+		if val, ok := metadata["localhost"]; ok {
```

Comment:
is localhost intentional?

Reply:
Yes. I'm OK for removing it. It is the logic for debugging. The reason why I add this is, for the rabbit mq testing with local debugging, we need to change the scaler code or do like this. (any other ideas are welcome) however, if we change the code every time for debugging, it could make mistake. This approach doesn't need to change the code. Just YAML side change required and not the breaking change for the current yaml files.

Reply:
@ahmelsayed 
Would you want to keep this or want to remove this?

Reply:
If you need any change in Rabbit MQ scaler, a separate PR for this is the best way to go to

Reply:
That's make sense. I create a separate PR for it.